src/lib-funnel/tagion/script/common.d

```
@@ -37,7 +37,7 @@ import tagion.utils.StdTime;
     @label("$in") const(DARTIndex)[] inputs; /// Hash pointer to input (DART)
     @label("$read") @optional @(filter.Initialized) const(DARTIndex)[] reads; /// Hash pointer to read-only input (DART)
     @label("$run") Document script; // Smart contract 
-    bool verify() {
+    bool verify() const pure nothrow @nogc {
```

Could we throw an error in case that the verify function is not called?